Core/DTO/CarCategoryDTO/CarCategoriesListDTO.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Linq;
+
+namespace Core.DTO.CarCategoryDTO
+{
+    public class CarCategoriesListDTO
+    {
+        public IQueryable<CarCategoryDTO> CarCategories { get; set; }
```

Comment:
Why IQueryable?

Reply:
Well, I agree, it will be better IEnumerable here.
[https://stackoverflow.com/questions/252785/what-is-the-difference-between-iqueryablet-and-ienumerablet](url)